### ISSUE TITLE
Sleep gesture in split doc unexpected (but is it wrong)

### ISSUE THREAD

**Issue by knox-dawson** [mb1986/rm-hacks#291]
In split doc, if I have the main doc at the bottom (e.g., so the dividing line is in the middle), I can swipe up near the DIVIDING LINE and put the tablet to sleep.

I recognize that this may not be wrong, but it was unexpected.  That is, I expected to have to swipe up near the top of the tablet display (where the split doc is currently displayed).

**Comment by RedTartan04** [comment]
Certainly "splitting" may need some explanation but I find it easier to understand the more both split and main behave the same. In this case it's good to be able to close only the split doc with that gesture and to stay in the main doc. It wont be logical if it was only possible if the split was in the upper half. Also other gestures like page turning or 1-finger-up-from-bottom work relative to the respective document, not relative to the physical screen.

**Comment by knox-dawson** [comment]
> Also other gestures like page turning or 1-finger-up-from-bottom work relative to the respective document window, not relative to the physical screen.

True.  So maybe the problem is more that swiping up near the top of the SPLIT doc (which is at the top of the tablet display) does NOT put the tablet to sleep.

**Comment by RedTartan04** [comment]
oh my, forget my last comment, I deleted it... I was distracted and read "close" where you wrote "sleep" :-/ For the sleep gesture I'm with you! It's a device-related gesture and should work only relative to the device, i.e. only on the top-edge, independent if the main or split doc is positioned there.

**Comment by atngames** [comment]
I fixed it, this will be okay in the next alpha (7)

**Comment by knox-dawson** [comment]
Similarly, swiping down from the top of the dividing line (like I might to open/close the toolbar) closes both docs.  I'll make another issue if you want.

**Comment by atngames** [comment]
Yeah since a fix already has been done, a new issue would be perfect
